modules/gdscript/tests/scripts/analyzer/features/infer_on_read_only.gd

```diff
@@ -0,0 +1,10 @@
+class A:
+	var x: String:
+		get:
+			return 'x'
+
+func test():
+	var a := A.new()
+	var x := a.x
+	x = 'ok'
+	print(x)
```

Comment:
This not really testing the issue that was fixed though. Missing a setter does not make the variable read-only, so it is not testing the change made.

The issue was about read-only properties in native classes.

Reply:
Ah, true, I learned about it only here after making the PR. 

Any suggestions for read-only props in native classes available in tests?

Reply:
`get_tree().root` should be one I think.

Reply:
I don't think there is tree in gdscript tests. If there is, how to get it?

Reply:
Ok, removed the test, now this is purely one-line addition. I'll mark this as resolved, but if there is some read-only property available in tests I will unresolve this.